test/GNNGraphs/transform.jl

```diff
@@ -268,3 +268,9 @@ end end
     @test nv(DG) == g.num_nodes
     @test ne(DG) == g.num_edges
 end
+
+@testset "RandomWalkPE" begin
+    g = rand_graph(10, 20, graph_type = GRAPH_T)
+    add_RandomWalkPE!(g, 3)
+    @test size(g.ndata.p) == (3, 10)
```

Comment:
we should have a test on a very small graph where the output can be manually computed (or we can compare it to the output from PyG).